wolfcrypt/src/rsa.c

```diff
@@ -3517,6 +3533,18 @@ static int RsaPrivateDecryptEx(const byte* in, word32 inLen, byte* out,
             }
             return ret;
         }
+    #elif defined(WOLFSSL_RENESAS_SCEPROTECT_CRYPTONLY)
+           #ifdef WOLF_CRYPTO_CB
+            if (key->devId != INVALID_DEVID) {
+                ret = wc_CryptoCb_Rsa(in, inLen, out,
+                            outLen, rsa_type, key, rng);
+                if (ret != CRYPTOCB_UNAVAILABLE)
+                    return ret;
```

Comment:
Can we add a debug message here letting users know that fall through happened?

Reply:
The same as above